src/main/java/baritone/process/BuilderProcess.java

```diff
@@ -1063,9 +1077,6 @@ private boolean valid(BlockState current, BlockState desired, boolean itemVerify
         if (desired.getBlock() instanceof AirBlock && Baritone.settings().buildIgnoreBlocks.value.contains(current.getBlock())) {
             return true;
         }
-        if (!(current.getBlock() instanceof AirBlock) && Baritone.settings().buildIgnoreExisting.value && !itemVerify) {
-            return true;
-        }
```

Comment:
Please don't break `buildIgnoreExisting` again.

Reply:
ah sorry this all happened because i tested it it with 1.20.1 build will fix that